config/jobs/kubernetes/sig-network/sig-network-misc.yaml

```diff
@@ -1,5 +1,56 @@
 presubmits:
   kubernetes/kubernetes:
+  - name: pull-kubernetes-e2e-ubuntu-gce-network-policies
+    always_run: false
+    run_if_changed: '^test/e2e/network/'
```

Comment:
Can we just enable network policy in the default presubmit clusters?

Reply:
I think that is risky, let it soak first, I don't know if that was tested before